controller/coq_lsp.ml

```diff
@@ -41,30 +41,13 @@ let option_default x d =
   | Some x -> x
 
 let oint_field name dict = option_cata U.to_int 0 List.(assoc_opt name dict)
-let ostring_field name dict = Option.map U.to_string (List.assoc_opt name dict)
 
 let odict_field name dict =
   option_default
     U.(to_option to_assoc (option_default List.(assoc_opt name dict) `Null))
     []
 
-module TraceValue = struct
-  type t =
-    | Off
-    | Messages
-    | Verbose
-
-  let parse = function
-    | "messages" -> Messages
-    | "verbose" -> Verbose
-    | "off" -> Off
-    | _ -> raise (Invalid_argument "TraceValue.parse")
-
-  let to_string = function
-    | Off -> "off"
-    | Messages -> "messages"
-    | Verbose -> "verbose"
-end
+let ostring_field name dict = Option.map U.to_string (List.assoc_opt name dict)
```

Comment:
Why move this?

Reply:
... because we need to read the value of `trace_value` in io.ml.

Reply:
Right but you moved it from line 44 to 51 I was curious why.

Reply:
Fixed now.